src/main/kotlin/uk/gov/justice/digital/assessments/restclient/DeliusIntegrationRestClient.kt

```diff
@@ -17,6 +18,7 @@ class DeliusIntegrationRestClient {
   @Qualifier("deliusIntegrationClient")
   internal lateinit var webClient: AuthenticatingRestClient
 
+  @Cacheable("caseDetails")
```

Comment:
I'm not sure how necessary this will be, but let's check the duration of the cache

Reply:
This was due to registrations being called on each page load, having this means we cache when the assessment is created and then read from the cache for registrations. They're not persisted in the DB AFAIK. Happy to discuss a sensible duration